.github/workflows/update-readme-txt/action.yml

```diff
@@ -8,14 +8,17 @@ inputs:
   PACKAGE_VERSION:
     required: true
     description: 'The package version to get the version from.'
+  README_TXT_PATH:
```

Comment:
As far as I can tell, we use `update-readme-txt` from a single action - do we want this as a parameter?

Reply:
this is to keep pubishV2 in working state, because path is different for current workflow and V2. I'll revert it when we delete publishV2.